src/Discord.Net.Core/Entities/Users/IUser.cs

```diff
@@ -47,11 +47,11 @@ public interface IUser : ISnowflakeEntity, IMentionable, IPresence
         /// </returns>
         string GetDefaultAvatarUrl();
         /// <summary>
-        ///     Gets the per-username unique ID for this user.
+        ///     Gets the per-username unique ID for this user. This will return "0000" for users, who have migrated to new username system.
```

Comment:
```suggestion
        ///     Gets the per-username unique ID for this user. This will return "0000" for users who have migrated to new username system.
```